aptos-move/block-executor/src/executor.rs

```diff
@@ -529,7 +529,7 @@ where
         assert!(self.concurrency_level > 1, "Must use sequential execution");
 
         let versioned_cache = MVHashMap::new();
-        let shared_counter = AtomicU32::new(0);
+        let shared_counter = AtomicU64::new(0);
```

Comment:
I thought the id would be 32_bit_counter || [metadata such as shard number]. Are we going to have id = 64_bit_counter instead?

Reply:
I think 32_bit_counter || .. but the metadata is not there yet? Will do it with 32-bits and just cast to u64 for now then👍